src/core/server/saved_objects/serialization/serializer.test.ts

```diff
@@ -611,25 +647,50 @@ describe('#savedObjectToRaw', () => {
       const v1 = namespaceAgnosticSerializer.savedObjectToRaw({
         type: 'foo',
         namespaces: ['bar'],
-        attributes: {
-          bar: true,
-        },
+        attributes: { bar: true },
       } as any);
 
       const v2 = namespaceAgnosticSerializer.savedObjectToRaw({
         type: 'foo',
         namespaces: ['bar'],
-        attributes: {
-          bar: true,
-        },
+        attributes: { bar: true },
       } as any);
 
-      expect(v1._id).toMatch(/foo\:[\w-]+$/);
+      expect(v1._id).toMatch(/^foo\:[\w-]+$/);
       expect(v1._id).not.toEqual(v2._id);
     });
 
-    test(`doesn't specify _source.namespace`, () => {
+    test(`doesn't specify _source.namespaces`, () => {
       const actual = namespaceAgnosticSerializer.savedObjectToRaw({
+        type: 'foo',
+        namespaces: ['bar'],
+        attributes: {},
+      } as any);
+
+      expect(actual._source).not.toHaveProperty('namespaces');
+    });
+  });
+
+  describe('multi-namespace type with a namespace', () => {
```

Comment:
Not sure how I missed this set of test cases and the next one (`'multi-namespace type with namespaces'`), but I added them now.